src/main/java/com/google/codeu/servlets/AboutMeServlet.java

```diff
@@ -7,16 +7,65 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.User;
+
 /**
- * Responds with a hard-coded message for testing purposes.
+ * Handles fetching and saving user data.
  */
 @WebServlet("/about")
-public class AboutMeServlet extends HttpServlet{
-  
- @Override
- public void doGet(HttpServletRequest request, HttpServletResponse response)
-   throws IOException {
-  
-  response.getOutputStream().println("hello world");
- }
+public class AboutMeServlet extends HttpServlet {
+
+  private Datastore datastore;
+
+  @Override
+  public void init() {
+    datastore = new Datastore();
+  }
+
+  /**
+   * Responds with the "about me" section for a particular user.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+
+    response.setContentType("text/html");
+
+    String user = request.getParameter("user");
+
```

Comment:
nit: Remove empty line. Add empty lines where necessary to make the code readable. Does not look like it is necessary here.